packages/schema/Cargo.toml

```diff
@@ -16,5 +16,6 @@ thiserror = "1.0.13"
 
 [dev-dependencies]
 anyhow = "1.0.57"
+cosmwasm-std = "1.1.8"
```

Comment:
Here we need the internal dependency style with the `path` argument. Otherwise we get duplicate dependencies (see lockfile diff)

Reply:
Ah, my bad